style.css

```diff
@@ -29,32 +21,38 @@ body {
 header {
     display: flex;
     overflow: hidden;
+    max-height: 90px;
+    height: 11vh;
+    min-height: 70px;
     background-color: var(--azulEscuro);
     justify-content: flex-start;
-    align-items: left;
-    padding: 20px 5%;
+    align-items: center;
 }
 
 nav {
     background-color: var(--azulEscuro);
 }
 
 .logo {
-    display: flex;
-    padding: 0;
     cursor: pointer;
 }
 
 .navLinks {
+    display: flex;
     list-style: none;
 }
 
 .navLinks li{
+    padding: 3vw;
```

Comment:
vw para esse caso de padding pode não ser muito responsivo. Tenta usar algo como:
`.navLinks{
display:flex;
justify-content: space-around;
flex-wrap:wrap;
list-style:none;
}`
junto de um 
`nav{ width:100% }`
para ver oque vc acha. Assim vc garante o espaçamento correto em várias larguras e ele também quebra a linha quando for necessário :)